docs/lightning-network/watchtowers.md

```diff
@@ -0,0 +1,85 @@
+
+This implies the nodes need to remain online, they need to remain connected to the blockchain's P2P network and they need to keep their wallet unlocked in order to sign the required transactions.
+
+For routing nodes - that is, nodes that are meant to remain online routing payments between distant participants of the network - that is not an issue since remaining online and well-connected is part of their standard operating procedure.
+
+However, for _ephemeral_ or _episodic_ nodes - ones that come online simply to make or receive an eventual payment and then go offline again - this is a harder requirement to achieve.
+
+In fact, a large amount of end-users fall exactly into this category: users that open a channel and make a few payments a month. For these users, keeping their wallets online and unlocked for a significant amount of time is infeasible.
+
+The solution for these users is to use a Watchtower Service.
+
+## Watchtower Basics
+
+A Watchtower is an LN node that can watch the chain on behalf of a third party and can send an appropriate justice transaction if it detects a relevant breach of contract.
+
+Every time a channel state is updated, the client node contacts the Watchtower and provides it information about the just _revoked_ state: the relevant keys, signatures for the justice transaction and a partial copy of the hash of commitment transaction. The keys and signatures are encrypted using the full hash of the commitment transaction as key.
+
+The Watchtower will then take that information as a hint and will begin to watch the chain for any signs that the provided transaction is mined. If it detects the partial tx hash on a block, it will find out the full hash of the revoked commitment transaction and will be able to decrypt the justice data.
+
+At this moment it will re-assemble the justice tx and publish it on the network, returning any funds locked in the delayed output of the breached commitment (the so-called `to_local` output) to its rightful owner.
+
+### Limitations
+
+The current implementation of the Watchtower feature can only redeem funds from the actually committed channel funds (that is, from the outputs that determine the relative balances of the channel participants). 
+
+In-flight payments (i.e. HTLCs) aren't supported by the current watchtower protocol, therefore in case of breaches those funds will still be available for the breacher to redeem.
+
+For nodes relying on Watchtowers, it is therefore recommended not to have individual states with many outstanding payments. In other words, users that are relying on watchtowers should generally wait until most of their in-flight payments are completed before they attempt a new one.
+
+## Using Watchtowers
+
+CLI users may use the [`wtclient`](dcrlncli/wtclient) command to add, remove and query the state of the watchtowers in their `dcrlnd` node.
```

Comment:
Link doesnt work, target needs to be `dcrlncli/wtclient.md`

Reply:
Fixed and rebased.